integration-libs/digital-payments/schematics/add-digital-payments/index.ts

```diff
@@ -44,26 +45,21 @@ export function addDigitalPaymentsFeature(
 }
 
 function addDigitalPayments(options: SpartacusDigitalPaymentsOptions): Rule {
-  return addLibraryFeature(
-    { ...options, lazy: false },
-    {
-      folderName: DIGITAL_PAYMENTS_FOLDER_NAME,
-      moduleName: DIGITAL_PAYMENTS_MODULE_NAME,
-      featureModule: {
-        importPath: SPARTACUS_DIGITAL_PAYMENTS,
-        name: DIGITAL_PAYMENTS_MODULE,
-      },
-      i18n: {
-        resources: DIGITAL_PAYMENTS_TRANSLATIONS,
-        chunks: DIGITAL_PAYMENTS_TRANSLATION_CHUNKS_CONFIG,
-        importPath: SPARTACUS_DIGITAL_PAYMENTS,
-      },
-      dependencyManagement: {
-        featureName: CLI_DIGITAL_PAYMENTS_FEATURE,
-        featureDependencies: {
-          [SPARTACUS_CHECKOUT]: [CLI_CHECKOUT_BASE_FEATURE],
-        },
-      },
```

Comment:
We need to keep this part.
It instructs schematics to install the checkout lib (if not installed)